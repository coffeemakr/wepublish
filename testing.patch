diff --git a/libs/api/src/lib/db/article.ts b/libs/api/src/lib/db/article.ts
index 7359af6a..ea27be09 100644
--- a/libs/api/src/lib/db/article.ts
+++ b/libs/api/src/lib/db/article.ts
@@ -14,11 +14,11 @@ export interface ArticleData {
   readonly lead?: string | null
   readonly seoTitle?: string | null
   readonly slug?: string | null
-  readonly tags: string[]
+  readonly tags?: string[]
 
   readonly canonicalUrl?: string | null
 
-  readonly properties: MetadataProperty[]
+  readonly properties?: MetadataProperty[]
 
   readonly imageID?: string | null
   readonly authors: ArticleRevisionAuthor[]
@@ -107,7 +107,7 @@ export type ArticleRevisionWithRelations = PrismaArticleRevision & {
 
 export type ArticleWithRevisions = PrismaArticle & {
   draft: ArticleRevisionWithRelations | null
-  pending: ArticleRevisionWithRelations | null
+  pending?: ArticleRevisionWithRelations | null
   published: ArticleRevisionWithRelations | null
 }
 
diff --git a/libs/api/src/lib/db/page.ts b/libs/api/src/lib/db/page.ts
index fb73c289..6f0ed1b5 100644
--- a/libs/api/src/lib/db/page.ts
+++ b/libs/api/src/lib/db/page.ts
@@ -14,9 +14,9 @@ export interface PageData {
 
   readonly title: string
   readonly description?: string | null
-  readonly tags: string[]
+  readonly tags?: string[]
 
-  readonly properties: MetadataProperty[]
+  readonly properties?: MetadataProperty[]
 
   readonly imageID?: string | null
 
@@ -85,7 +85,7 @@ export type PageRevisionWithProperties = PrismaPageRevision & {
 
 export type PageWithRevisions = PrismaPage & {
   draft: PageRevisionWithProperties | null
-  pending: PageRevisionWithProperties | null
+  pending?: PageRevisionWithProperties | null
   published: PageRevisionWithProperties | null
 }
 
diff --git a/libs/api/src/lib/graphql/article.ts b/libs/api/src/lib/graphql/article.ts
index 8d80fb27..77357e01 100644
--- a/libs/api/src/lib/graphql/article.ts
+++ b/libs/api/src/lib/graphql/article.ts
@@ -275,7 +275,7 @@ export const GraphQLPublicArticle: GraphQLObjectType<PublicArticle, Context> =
         type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLMetadataPropertyPublic))),
         resolve: ({properties}) => {
           return properties
-            .filter(property => property.public)
+            ?.filter(property => property.public)
             .map(({key, value}) => ({key, value}))
         }
       },
diff --git a/libs/api/src/lib/graphql/article/article.private-mutation.ts b/libs/api/src/lib/graphql/article/article.private-mutation.ts
index 43a6a3ee..049c7e54 100644
--- a/libs/api/src/lib/graphql/article/article.private-mutation.ts
+++ b/libs/api/src/lib/graphql/article/article.private-mutation.ts
@@ -178,7 +178,8 @@ export const duplicateArticle = async (
       createMany: {
         data: socialMediaAuthors.map(({authorId}) => ({authorId}))
       }
-    }
+    },
+    blocks: []
   }
 
   return articleClient.create({
@@ -303,7 +304,8 @@ export const unpublishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -333,7 +335,8 @@ export const unpublishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           }
         }
       },
@@ -491,7 +494,8 @@ export const publishArticle = async (
                 createMany: {
                   data: socialMediaAuthors.map(({authorId}) => ({authorId}))
                 }
-              }
+              },
+              blocks: []
             },
             update: {
               ...revision,
@@ -521,7 +525,8 @@ export const publishArticle = async (
                 createMany: {
                   data: socialMediaAuthors.map(({authorId}) => ({authorId}))
                 }
-              }
+              },
+              blocks: []
             }
           }
         },
@@ -579,7 +584,8 @@ export const publishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -609,7 +615,8 @@ export const publishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           }
         }
       },
diff --git a/libs/api/src/lib/graphql/page.ts b/libs/api/src/lib/graphql/page.ts
index 56d380f0..6b63518e 100644
--- a/libs/api/src/lib/graphql/page.ts
+++ b/libs/api/src/lib/graphql/page.ts
@@ -210,7 +210,7 @@ export const GraphQLPublicPage = new GraphQLObjectType<PublicPage, Context>({
     properties: {
       type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLMetadataPropertyPublic))),
       resolve: ({properties}) => {
-        return properties.filter(property => property.public).map(({key, value}) => ({key, value}))
+        return properties?.filter(property => property.public).map(({key, value}) => ({key, value}))
       }
     },
 
diff --git a/libs/api/src/lib/graphql/page/page.private-mutation.ts b/libs/api/src/lib/graphql/page/page.private-mutation.ts
index 8d21a995..456df84e 100644
--- a/libs/api/src/lib/graphql/page/page.private-mutation.ts
+++ b/libs/api/src/lib/graphql/page/page.private-mutation.ts
@@ -140,7 +140,8 @@ export const duplicatePage = async (
       createMany: {
         data: duplicatedProperties
       }
-    }
+    },
+    blocks: []
   }
 
   return pageClient.create({
@@ -236,7 +237,8 @@ export const unpublishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -250,7 +252,8 @@ export const unpublishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           }
         }
       },
@@ -387,7 +390,8 @@ export const publishPage = async (
                 createMany: {
                   data: properties
                 }
-              }
+              },
+              blocks: []
             },
             update: {
               ...revision,
@@ -401,7 +405,8 @@ export const publishPage = async (
                 createMany: {
                   data: properties
                 }
-              }
+              },
+              blocks: []
             }
           }
         },
@@ -443,7 +448,8 @@ export const publishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -454,7 +460,8 @@ export const publishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           }
         }
       },
