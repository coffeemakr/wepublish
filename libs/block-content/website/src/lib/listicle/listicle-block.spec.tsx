import {runStorybookTests} from '@wepublish/testing'
import * as stories from './listicle-block.stories'

describe('Listicle Block', () => {
  runStorybookTests(stories)
})
