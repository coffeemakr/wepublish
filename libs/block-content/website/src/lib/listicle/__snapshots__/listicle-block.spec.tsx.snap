// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listicle Block should render Default 1`] = `
<DocumentFragment>
  <div
    class="css-5ednon e1iakh080"
  >
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        1
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        2
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        3
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        4
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Listicle Block should render WithClassName 1`] = `
<DocumentFragment>
  <div
    class="extra-classname css-5ednon e1iakh080"
  >
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        1
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        2
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        3
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        4
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Listicle Block should render WithEmotion 1`] = `
<DocumentFragment>
  <div
    class="css-5ednon e1iakh080"
  >
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        1
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        2
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        3
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        4
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Listicle Block should render WithoutImage 1`] = `
<DocumentFragment>
  <div
    class="css-5ednon e1iakh080"
  >
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        1
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        2
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        3
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <img
        alt="Dicta itaque nisi mollitia voluptate."
        class="css-1a2v9hb e1nfkf2e0"
        height="500"
        loading="lazy"
        srcset="https://picsum.photos/seed/8xHsH/800/400 800w,
https://picsum.photos/seed/dkVOrEU/500/300 500w,
https://picsum.photos/seed/3ZbG2/300/200 300w,
https://picsum.photos/seed/FUAYxo2gyq/200/100 200w"
        title="aliquid deleniti quisquam"
        width="1000"
      />
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1egb6s2 e1iakh081"
    >
      <div
        class="css-1lfpiq0 e1iakh082"
      >
        4
      </div>
      <h3
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        Foobar
      </h3>
      <div
        class="css-0 e1e9qmuq0"
      >
        <div
          data-gramm="false"
          data-slate-editor="true"
          data-slate-node="value"
          style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
            data-slate-node="element"
          >
            <span
              data-slate-node="text"
            >
              <span
                data-slate-string="true"
              >
                Nisi laudantium nihil ex praesentium repellat tempora beatae fugit minima.
              </span>
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
